packaging/functions.sh

```diff
@@ -253,10 +253,11 @@ set_mod_version() {
 #   Makefile (install-linux-shortcuts target for local installs and downstream packaging)
 install_linux_shortcuts() {
 	SRC_PATH="${1}"
-	OPENRA_PATH="${2}"
```

Comment:
The comments above will need to be updated to match.

Reply:
Also updated the `shift` command.